modules/olmv1-catalog-queries.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies:
+//
+// * operators/olm_v1/olmv1-installing-an-operator-from-a-catalog.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="olmv1-catalog-queries_{context}"]
+= Common catalog queries
+
+You can query catalogs by using the `jq` CLI tool.
+
+// https://docs.asciidoctor.org/asciidoc/latest/tables/format-cell-content/#a-operator
+// https://docs.asciidoctor.org/asciidoc/latest/tables/data-format/#escape-the-cell-separator
+.Common catalog queries
+[options="header",cols="1,3"]
+|===
+|Query |Request
+
+|Packages
+a|
+[source,terminal]
+----
+$ jq -s '.[] \| select( .schema == "olm.package") \| \
+  .name' <catalog_name>.json
+----
+
+|Package metadata
+a|
+[source,terminal]
+----
+$ jq -s '.[] \| select( .schema == "olm.package") \| \
+  select( .name == "<package_name>")' <catalog_name>.json
+----
+
+|Catalog blobs
```

Comment:
"Catalog blobs" could be a bit misleading since this query is all blobs in the catalog for a particular package. Maybe something more along the lines of "Catalog blobs for package" ?